libkpod/container_server.go

```diff
@@ -13,18 +16,24 @@ import (
 	"github.com/docker/docker/pkg/truncindex"
 	"github.com/kubernetes-incubator/cri-o/libkpod/sandbox"
 	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/pkg/annotations"
 	"github.com/kubernetes-incubator/cri-o/pkg/storage"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/selinux/go-selinux/label"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
 // ContainerServer implements the ImageServer
 type ContainerServer struct {
-	runtime            *oci.Runtime
-	store              cstorage.Store
-	storageImageServer storage.ImageServer
-	ctrNameIndex       *registrar.Registrar
-	ctrIDIndex         *truncindex.TruncIndex
-	podNameIndex       *registrar.Registrar
-	podIDIndex         *truncindex.TruncIndex
+	runtime              *oci.Runtime
+	store                cstorage.Store
+	storageImageServer   storage.ImageServer
+	storageRuntimeServer storage.RuntimeServer // Should this go back to server once shared state gets merged?
```

Comment:
No, we're going to need to keep this in ContainerServer - it's necessary for setting up storage in create